options.c

```diff
@@ -76,6 +76,7 @@ int smart_artist_sort = 1;
 int scroll_offset = 2;
 int rewind_offset = 5;
 int skip_track_info = 0;
+int ignore_duplicates = 1;
```

Comment:
@flyingmutant, @nefthy, do you think this should this be enabled by default? Personally, the only potential issue I see is confusion if an added track doesn't appear without an explanation (especially if the user tries to add a specific track).

Also, depending on how strict we are about breaking changes in releases, this might be considered to be a breaking change since it changes the behavior when adding tracks. However, this won't affect existing libraries, so it could also be argued the other way around.

Reply:
Actually the same code is used when lib.pl is being imported each startup, so it will remove dups from existing libraries if left 
 enabled by default.

Reply:
Looks like I missed that since I was testing with an existing rc with this disable previously. In that case, we should probably leave this disabled until maybe the next major version or after we have some more feedback on how this is used.